src/Symfony/Component/Config/Resource/ReflectionClassResource.php

```diff
@@ -157,12 +157,16 @@ private function generateSignature(\ReflectionClass $class)
 
         if ($class->isSubclassOf(EventSubscriberInterface::class)) {
             yield EventSubscriberInterface::class;
-            yield print_r(\call_user_func(array($class->name, 'getSubscribedEvents')), true);
+            if (!$class->isAbstract()) {
```

Comment:
the check should be merged in the above "if", no need to yield `EventSubscriberInterface::class`
(same below, and I agree with @chalasr comment below also)

Reply:
or, just before the current two checks, we could just add
```
if ($class->isAbstract() || $class->isInterface()) {
    return;
}
```